metrics/serve.go

```diff
@@ -72,11 +76,8 @@ func RunMetrics(cfg Config, stop chan struct{}) error {
 	go func() {
 		for tick := range cycleTick.C {
 			log.Infof("%v: refreshing cycle", tick)
-			metricsMux.Lock()
-			deleteValues()
 			cycleId++
```

Comment:
Do you need to cycleId++ also inside the cycleValues as it needs to be protected with the same mutex.

Reply:
Done.